onnxruntime/core/providers/tensorrt/tensorrt_execution_provider_helper.cc

```diff
@@ -3,10 +3,34 @@
 
 #include "core/providers/shared_library/provider_api.h"
 #include "tensorrt_execution_provider.h"
+#include "core/framework/murmurhash3.h"
 #include <iostream>
 
 namespace onnxruntime {
 
+// Get unique graph name based on graph's name and all nodes' name
+std::string GetUniqueGraphName(const Graph& graph) {
```

Comment:
this helper is only used in this file right?  maybe put it in anonymous namespace?

Reply:
done